### ISSUE TITLE
clog in both cpuinfo and nnpack... 

### ISSUE THREAD

**Issue by jiapei100** [pytorch/cpuinfo#4]
If I install one package, I''ll fail the other with the following **ERROR** message:

```
Unpacking cpuinfo (20180706-6) ...
dpkg: error processing archive ....../cpuinfo/build/cpuinfo_20180706-6_amd64.deb (--install):
 trying to overwrite '/usr/local/include/clog.h', which is also in package nnpack 20180706-6
dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)
Errors were encountered while processing:
```

Any suggestions?

**Comment by Maratyszcza** [comment]
NNPACK and cpuinfo are, generally, not expected to be installed through system packaging mechanisms, instead, they are designed to be imported via `add_subdirectory` in CMake and statically linked to projects using them, e.g. PyTorch.

**Comment by jiapei100** [comment]
@Maratyszcza Should I install Caffe2 independently? 

It seems Caffe2 is NOW integrated with PyTorch, should I build PyTorch directly?
But, I'm using Ubuntu 18.04 with GCC 7.3.0, I failed to build current PyTorch...

Any other suggestions?

**Comment by cdluminate** [comment]
@jiapei100 May I ask where is that .deb package from? I'm going to package pytorch for Debian.

**Comment by Maratyszcza** [comment]
Fixed in #129
